apis/youtube.py

```diff
@@ -38,19 +38,23 @@ def search(self, search_query, max_length, min_view_count):
     def download(self, url, audio_bitrate):
 
         youtube_video = pytubeYouTube(url)
-        youtube_video_streams = youtube_video.streams.filter(only_audio=True, file_extension='mp4')
+        youtube_video_streams = youtube_video.streams.filter(only_audio=True)
```

Comment:
@ImAccia I found that adding file_existion='mp4' increased availability of higher audio qualities, but needs confirming